cmd/nodes.go

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2019 Humio Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func newNodesCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "nodes",
+		Short: "Manage nodes",
```

Comment:
Maybe you should make the distinction between `nodes` and `cluster` commands clear.
I was confused at first since in my mind "nodes" is part of cluster management.

The least you could do would maybe be include a bit more description in the "Short" field for both. Alternatively nest the `nodes` command under `cluster`

Reply:
What do you think?

Reply:
Good point. I think I'll move `nodes` to a nested command under `cluster`.